src/app_server.rs

```diff
@@ -632,7 +632,14 @@ impl AppServer {
                 .stdin(Stdio::piped())
                 .args(&owg.extra_params)
                 .spawn()?;
-            b64_writer(child.stdin.unwrap()).write_all(key.secret())?;
+            b64_writer(child.stdin.take().unwrap()).write_all(key.secret())?;
+            let status = child.wait()?;
```

Comment:
This is dangerous, now timing of our event loop is distracted by the `wg` command.